### ISSUE TITLE
Publish (collection) repository on npm

### ISSUE THREAD

**Issue by alcferreira** [CraveFood/farmblocks#18]

**Comment by fczuardi** [comment]
after #3 we will publish each individual component to npm, and if we want to publish a collection of all blocks under a ``@crave/farmblocks`` package, that package can be a package in the monorepo packages subdir as well...

so we can keep this issue opened to track that task (the publishing of the collection package) or close this as wontfix, either way is fine by me
